### DISCUSSION TITLE
Beacon "cheater" feature would be really nice.

### DISCUSSION THREAD

**Discussion by smellyonionman** [Zrips/CMI#784] [Feature Requests]
CMI feels like the right plugin to include such a feature.. basically, I'd like to power beacons without pyramids.  They're just so awful to hide!

Sometimes all you want is a nice, bright beam to light the way.  And maybe a status effect or two.  But mainly the aesthetic light beam is what I personally want.  Can't stand those pyramids, to be perfectly honest.

**Comment by CloudeLecaw** [comment]
Nice idea, but it should be also an admin feature (only useable with permission) else u will lightbeams everywhere 😂
